shared/constants/swaps.js

```diff
@@ -46,7 +46,7 @@ const BSC_CONTRACT_ADDRESS = '0x1a1ec25DC08e98e5E93F1104B5e5cdD298707d31';
 const METASWAP_ETH_API_HOST = 'https://api.metaswap.codefi.network';
 
 const METASWAP_BNB_API_HOST = 'https://bsc-api.metaswap.codefi.network';
-const BNB_CHAIN_ID = '0x38';
+export const BNB_CHAIN_ID = '0x38';
```

Comment:
This seems better suited for the `network` constants module - that's where we have all of our other chain IDs. Especially since it's now being used for something unrelated to swaps.

Reply:
Also, why is this called `BNB_CHAIN_ID`? Isn't the chain called "BSC"? cc @danjm

Reply:
ack, BNB is the symbol of the default token on the chain, so this is a mistake

Reply:
You could add the export to the network constants module, as well as a `CHAIN_ID_TOKEN_IMAGE_MAP` export, and then leave the `constants/swaps` file unchanged. We can then address redundancies in a later PR.

Reply:
I brought up in a [different comment](https://github.com/MetaMask/metamask-extension/pull/10777#discussion_r604868465) that this image corresponds with the native currency, not the chain ID. Different chains with the same currency will want the same image here (e.g. Mainnet and the Ethereum testnets)

Reply:
Ah, sorry, yes. So a `NATIVE_CURRENCY_TOKEN_IMAGE_MAP` is what we want here.

Reply:
Implemented `NATIVE_CURRENCY_TOKEN_IMAGE_MAP`.